packages/aws-combo/package.json

```diff
@@ -22,28 +22,22 @@
     "url": "git+https://github.com/mvayngrib/aws-utils.git"
   },
   "scripts": {
-    "test": "echo \"Error: run tests from root\" && exit 1",
+    "test": "ts-node ./src/__tests__/index.ts",
+    "test:ci": "node ./lib/__tests__/index.js",
     "tsc": "tsc"
   },
   "bugs": {
     "url": "https://github.com/mvayngrib/aws-utils/issues"
   },
   "dependencies": {
-    "@tradle/aws-client-factory": "^1.0.1-alpha.17",
-    "@tradle/aws-cloudformation-client": "^1.0.1-alpha.17",
-    "@tradle/aws-cloudwatch-client": "^1.0.1-alpha.17",
-    "@tradle/aws-common-utils": "^1.0.1-alpha.17",
-    "@tradle/aws-iam-client": "^1.0.1-alpha.17",
-    "@tradle/aws-lambda-client": "^1.0.1-alpha.17",
-    "@tradle/aws-s3-client": "^1.0.1-alpha.17",
-    "@tradle/aws-sns-client": "^1.0.1-alpha.17",
+    "@tradle/aws-client-factory": "file:../aws-client-factory",
+    "@tradle/aws-cloudformation-client": "file:../aws-cloudformation-client",
+    "@tradle/aws-cloudwatch-client": "file:../aws-cloudwatch-client",
+    "@tradle/aws-common-utils": "file:../aws-common-utils",
+    "@tradle/aws-iam-client": "file:../aws-iam-client",
+    "@tradle/aws-lambda-client": "file:../aws-lambda-client",
+    "@tradle/aws-s3-client": "file:../aws-s3-client",
+    "@tradle/aws-sns-client": "file:../aws-sns-client",
```

Comment:
how does resolution work when aws-combo is installed in an external project (e.g. tradle/mycloud)?

Reply:
It is my understanding that lerna will replace these links on publication.

Reply:
i didn't know that, cool!

Reply:
https://github.com/lerna/lerna/blob/a47fc294393a3e9507a8207a5a2f07648a524722/commands/publish/index.js#L529